templates/expo-template-bare-minimum/android/app/proguard-rules.pro

```diff
@@ -7,4 +7,8 @@
 # For more details, see
 #   http://developer.android.com/guide/developing/tools/proguard.html
 
+# react-native-reanimated
+-keep class com.swmansion.reanimated.** { *; }
+-keep class com.facebook.react.turbomodule.** { *; }
```

Comment:
was this intentional for this pr?

Reply:
yes, i've seen people forget to add this from [the installation guide](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation#proguard) and needs a config-plugin to add it. as reanimated is popular, i think it's no hurt to add these.

ideally, reanimated should add the proguard rules from [`consumerProguardFiles`](https://developer.android.com/studio/projects/android-library#Considerations). i can propose a pr to them and remove our builtin reanimated rules in next sdk cycle.

Reply:
Does this change mean reanimated is now required for building on Android again?

Reply:
no, it's just these java classes will not be obfuscated. if reanimated is not installed, only `com.facebook.react.turbomodule.**` class will be kept.